RELEASES.md

```diff
@@ -35,7 +35,7 @@
 - Fix: publish latest Notion version via self-hosted endpoint (#230)
 - Fix: eliminate excessive fetching and scanning for exact versions (#227)
 - Rename `notion use` to `notion pin` (#226)
-- Base filesystem isolation on `NOTION_HOME` env var (#224)
+- Base filesystem isolation on `notion_HOME` env var (#224)
```

Comment:
Case was changed.

Reply:
Doh. Thank you; pushed a fix in 378411f2.